_docs/for-contributors/2017-01-01-building-from-source.md

```diff
@@ -55,9 +87,9 @@ It is suggested having at least 8GB of RAM and some swap space for the build pro
 
 After the project is built - it can take quite a long time -  the built binaries can be launched using the `stack exec` command. Let's discuss important binaries briefly before proceeding to the next step.
 
-### Build Commands without Nix
+## Pure Stack-based build (not recommended)
```

Comment:
Another Pure reference to be removed.

Reply:
Nuked the entire section from the orbit.